core/src/epicli/data/common/ansible/playbooks/postgres_exporter.yml

```diff
@@ -0,0 +1,6 @@
+---
+- hosts: postgres_exporter[0]
```

Comment:
I think we need to handle also clustered postgres, we should iterate over the list.